Move VS Code plugin, add IntelliJ, add APIsecurity.io newsletter (#67)
yosriady/api-development-tools: README.md

- [Swagger Inspector](https://swagger.io/tools/swagger-inspector/): Test and auto-generate OpenAPI documentation for any API.
- [Swagger Editor](http://editor.swagger.io/): An editor for designing Swagger specifications.
- [Swagger Tools and Integrations](https://swagger.io/open-source-integrations/): A list of libraries and frameworks serving the Swagger ecosystem.

> - [OpenAPI extension for Microsoft Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi): Microsoft VS Code IDE has an OpenAPI extension that includes OpenAPI / Swagger file creation, HTML preview, editing, navigation, code snippets, IntelliSense, security static analysis.

Reviewer: ```suggestion
- [OpenAPI extension for VS Code](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi):  Visual Studio Code (VS Code) extension that provides support for the OpenAPI Specification.
```

> - [OpenAPI extension for JetBrains IDEs: IntelliJ, PyCharm, etc](https://plugins.jetbrains.com/plugin/14837-openapi-swagger-editor): This IDE plugin for JetBrains includes OpenAPI / Swagger file creation, HTML preview, editing, navigation, code snippets, IntelliSense, security static analysis.

Reviewer: ```suggestion
- [OpenAPI plugin for JetBrains IDEs](https://plugins.jetbrains.com/plugin/14837-openapi-swagger-editor): Jetbrains plugin that provides support for the OpenAPI Specification.
```

- [JSON-LD](https://json-ld.org/)

## Learning Resources

> - [APIsecurity.io weekly newsletter and encyclopedia on API Security](https://apisecurity.io)

Reviewer: Can we move this to the bottom of the list?

Author: Sure, feel free to order as you see fit
